src/nyx/features/radial_distribution.cpp

```diff
@@ -44,6 +44,11 @@ void RadialDistributionFeature::calculate(LR& r)
 	// Get ahold of the center pixel
 	const Pixel2& pxO = raw_pixels[idxO];
 
+	//
+	// **** Version 1 (slower and accurate)
+	//
+#if 0

```

Comment:
dead code :-)

Reply:
I need to keep the old algorithm somewhere. Source control (Git) is a non-workable option as I'd be digging important changes out of a pile of trash ones.